crates/sui-protocol-config/src/lib.rs

```diff
@@ -201,6 +201,7 @@ const MAX_PROTOCOL_VERSION: u64 = 70;
 // Version 70: Enable smart ancestor selection in testnet.
 //             Enable probing for accepted rounds in round prober in testnet
 //             Add new gas model version to update charging of native functions.
+//             Framework update to improve gas/wall time efficiency of some Move stdlib vector functions
```

Comment:
You won't be able to land this under 70 after the branch cut, and we will need to bump the version here. 
I'm trying to cherry pick #19894 for Walrus into 70. So we can either roll this into that and cherry-pick it too, or wait until #19894 lands